packages/maskbook/src/extension/options-page/DashboardRouters/Wallets.tsx

```diff
@@ -84,6 +85,11 @@ const useStyles = makeStyles((theme) =>
     }),
 )
 
+export const DashboardWalletsContext = createContext<{
```

Comment:
Add context here in `Wallets.tsx` rather than integrate context with  `useAssetsDetailedCallback.ts`, since only in this component props need to be passed through too many components.